lib/commands/new.js

```diff
@@ -55,6 +55,10 @@ module.exports = Command.extend({
       return Promise.reject(new SilentError(message));
     }
 
+    if (commandOptions.blueprint[0] === '.') {
+      commandOptions.blueprint = path.resolve(process.cwd(), commandOptions.blueprint);
```

Comment:
that being said, can we extract this to a helper function that is shared between new and init?